{
	"document": {
		"aggregate_severity": {
			"namespace": "https://www.suse.com/support/security/rating/",
			"text": "moderate"
		},
		"category": "csaf_vex",
		"csaf_version": "2.0",
		"distribution": {
			"text": "Copyright 2024 SUSE LLC. All rights reserved.",
			"tlp": {
				"label": "WHITE",
				"url": "https://www.first.org/tlp/"
			}
		},
		"lang": "en",
		"notes": [
			{
				"category": "summary",
				"text": "SUSE CVE-2018-16984",
				"title": "Title"
			},
			{
				"category": "description",
				"text": "An issue was discovered in Django 2.1 before 2.1.2, in which unprivileged users can read the password hashes of arbitrary accounts. The read-only password widget used by the Django Admin to display an obfuscated password hash was bypassed if a user has only the \"view\" permission (new in Django 2.1), resulting in display of the entire password hash to those users. This may result in a vulnerability for sites with legacy user accounts using insecure hashes.",
				"title": "Description of the CVE"
			},
			{
				"category": "legal_disclaimer",
				"text": "CSAF 2.0 data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
				"title": "Terms of use"
			}
		],
		"publisher": {
			"category": "vendor",
			"contact_details": "https://www.suse.com/support/security/contact/",
			"name": "SUSE Product Security Team",
			"namespace": "https://www.suse.com/"
		},
		"references": [
			{
				"category": "external",
				"summary": "CVE-2018-16984",
				"url": "https://www.suse.com/security/cve/CVE-2018-16984"
			},
			{
				"category": "external",
				"summary": "SUSE Security Ratings",
				"url": "https://www.suse.com/support/security/rating/"
			},
			{
				"category": "external",
				"summary": "SUSE Bug 1109621 for CVE-2018-16984",
				"url": "https://bugzilla.suse.com/1109621"
			}
		],
		"title": "SUSE CVE CVE-2018-16984",
		"tracking": {
			"current_release_date": "2024-07-20T04:10:27Z",
			"generator": {
				"date": "2023-02-15T04:23:46Z",
				"engine": {
					"name": "cve-database.git:bin/generate-csaf-vex.pl",
					"version": "1"
				}
			},
			"id": "CVE-2018-16984",
			"initial_release_date": "2023-02-15T04:23:46Z",
			"revision_history": [
				{
					"date": "2023-02-15T04:23:46Z",
					"number": "2",
					"summary": "Current version"
				},
				{
					"date": "2024-07-20T04:10:27Z",
					"number": "3",
					"summary": "Current version"
				}
			],
			"status": "interim",
			"version": "3"
		}
	},
	"product_tree": {
		"branches": [
			{
				"branches": [
					{
						"branches": [
							{
								"category": "product_name",
								"name": "SUSE Enterprise Storage 4",
								"product": {
									"name": "SUSE Enterprise Storage 4",
									"product_id": "SUSE Enterprise Storage 4",
									"product_identification_helper": {
										"cpe": "cpe:/o:suse:ses:4"
									}
								}
							},
							{
								"category": "product_name",
								"name": "SUSE Enterprise Storage 5",
								"product": {
									"name": "SUSE Enterprise Storage 5",
									"product_id": "SUSE Enterprise Storage 5",
									"product_identification_helper": {
										"cpe": "cpe:/o:suse:ses:5"
									}
								}
							},
							{
								"category": "product_name",
								"name": "openSUSE Tumbleweed",
								"product": {
									"name": "openSUSE Tumbleweed",
									"product_id": "openSUSE Tumbleweed",
									"product_identification_helper": {
										"cpe": "cpe:/o:opensuse:tumbleweed"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python-Django",
								"product": {
									"name": "python-Django",
									"product_id": "python-Django",
									"product_identification_helper": {
										"cpe": "cpe:2.3:a:djangoproject:django:*:*:*:*:*:*:*:*"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python310-Django-4.2.11-2.1",
								"product": {
									"name": "python310-Django-4.2.11-2.1",
									"product_id": "python310-Django-4.2.11-2.1",
									"product_identification_helper": {
										"cpe": "cpe:2.3:a:djangoproject:django:4.2.11:*:*:*:*:*:*:*"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python310-Django4-4.2.14-1.1",
								"product": {
									"name": "python310-Django4-4.2.14-1.1",
									"product_id": "python310-Django4-4.2.14-1.1"
								}
							},
							{
								"category": "product_version",
								"name": "python311-Django-4.2.11-2.1",
								"product": {
									"name": "python311-Django-4.2.11-2.1",
									"product_id": "python311-Django-4.2.11-2.1",
									"product_identification_helper": {
										"cpe": "cpe:2.3:a:djangoproject:django:4.2.11:*:*:*:*:*:*:*"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python311-Django4-4.2.14-1.1",
								"product": {
									"name": "python311-Django4-4.2.14-1.1",
									"product_id": "python311-Django4-4.2.14-1.1"
								}
							},
							{
								"category": "product_version",
								"name": "python312-Django-4.2.11-2.1",
								"product": {
									"name": "python312-Django-4.2.11-2.1",
									"product_id": "python312-Django-4.2.11-2.1",
									"product_identification_helper": {
										"cpe": "cpe:2.3:a:djangoproject:django:4.2.11:*:*:*:*:*:*:*"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python312-Django4-4.2.14-1.1",
								"product": {
									"name": "python312-Django4-4.2.14-1.1",
									"product_id": "python312-Django4-4.2.14-1.1"
								}
							},
							{
								"category": "product_version",
								"name": "python36-Django-3.2.7-2.3",
								"product": {
									"name": "python36-Django-3.2.7-2.3",
									"product_id": "python36-Django-3.2.7-2.3"
								}
							},
							{
								"category": "product_version",
								"name": "python38-Django-3.2.7-2.3",
								"product": {
									"name": "python38-Django-3.2.7-2.3",
									"product_id": "python38-Django-3.2.7-2.3",
									"product_identification_helper": {
										"cpe": "cpe:2.3:a:djangoproject:django:3.2.7:*:*:*:*:*:*:*"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python39-Django-3.2.7-2.3",
								"product": {
									"name": "python39-Django-3.2.7-2.3",
									"product_id": "python39-Django-3.2.7-2.3",
									"product_identification_helper": {
										"cpe": "cpe:2.3:a:djangoproject:django:3.2.7:*:*:*:*:*:*:*"
									}
								}
							}
						],
						"category": "product_family",
						"name": "SUSE Linux Enterprise"
					}
				],
				"category": "vendor",
				"name": "SUSE"
			}
		],
		"relationships": [
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python310-Django-4.2.11-2.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python310-Django-4.2.11-2.1"
				},
				"product_reference": "python310-Django-4.2.11-2.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python310-Django4-4.2.14-1.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python310-Django4-4.2.14-1.1"
				},
				"product_reference": "python310-Django4-4.2.14-1.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python311-Django-4.2.11-2.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python311-Django-4.2.11-2.1"
				},
				"product_reference": "python311-Django-4.2.11-2.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python311-Django4-4.2.14-1.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python311-Django4-4.2.14-1.1"
				},
				"product_reference": "python311-Django4-4.2.14-1.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python312-Django-4.2.11-2.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python312-Django-4.2.11-2.1"
				},
				"product_reference": "python312-Django-4.2.11-2.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python312-Django4-4.2.14-1.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python312-Django4-4.2.14-1.1"
				},
				"product_reference": "python312-Django4-4.2.14-1.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python36-Django-3.2.7-2.3 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python36-Django-3.2.7-2.3"
				},
				"product_reference": "python36-Django-3.2.7-2.3",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python38-Django-3.2.7-2.3 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python38-Django-3.2.7-2.3"
				},
				"product_reference": "python38-Django-3.2.7-2.3",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python39-Django-3.2.7-2.3 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python39-Django-3.2.7-2.3"
				},
				"product_reference": "python39-Django-3.2.7-2.3",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-Django as component of SUSE Enterprise Storage 4",
					"product_id": "SUSE Enterprise Storage 4:python-Django"
				},
				"product_reference": "python-Django",
				"relates_to_product_reference": "SUSE Enterprise Storage 4"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-Django as component of SUSE Enterprise Storage 5",
					"product_id": "SUSE Enterprise Storage 5:python-Django"
				},
				"product_reference": "python-Django",
				"relates_to_product_reference": "SUSE Enterprise Storage 5"
			}
		]
	},
	"vulnerabilities": [
		{
			"cve": "CVE-2018-16984",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2018-16984"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "An issue was discovered in Django 2.1 before 2.1.2, in which unprivileged users can read the password hashes of arbitrary accounts. The read-only password widget used by the Django Admin to display an obfuscated password hash was bypassed if a user has only the \"view\" permission (new in Django 2.1), resulting in display of the entire password hash to those users. This may result in a vulnerability for sites with legacy user accounts using insecure hashes.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"known_not_affected": [
					"SUSE Enterprise Storage 4:python-Django",
					"SUSE Enterprise Storage 5:python-Django"
				],
				"recommended": [
					"openSUSE Tumbleweed:python310-Django-4.2.11-2.1",
					"openSUSE Tumbleweed:python310-Django4-4.2.14-1.1",
					"openSUSE Tumbleweed:python311-Django-4.2.11-2.1",
					"openSUSE Tumbleweed:python311-Django4-4.2.14-1.1",
					"openSUSE Tumbleweed:python312-Django-4.2.11-2.1",
					"openSUSE Tumbleweed:python312-Django4-4.2.14-1.1",
					"openSUSE Tumbleweed:python36-Django-3.2.7-2.3",
					"openSUSE Tumbleweed:python38-Django-3.2.7-2.3",
					"openSUSE Tumbleweed:python39-Django-3.2.7-2.3"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2018-16984",
					"url": "https://www.suse.com/security/cve/CVE-2018-16984"
				},
				{
					"category": "external",
					"summary": "SUSE Security Ratings",
					"url": "https://www.suse.com/support/security/rating/"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1109621 for CVE-2018-16984",
					"url": "https://bugzilla.suse.com/1109621"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:python310-Django-4.2.11-2.1",
						"openSUSE Tumbleweed:python310-Django4-4.2.14-1.1",
						"openSUSE Tumbleweed:python311-Django-4.2.11-2.1",
						"openSUSE Tumbleweed:python311-Django4-4.2.14-1.1",
						"openSUSE Tumbleweed:python312-Django-4.2.11-2.1",
						"openSUSE Tumbleweed:python312-Django4-4.2.14-1.1",
						"openSUSE Tumbleweed:python36-Django-3.2.7-2.3",
						"openSUSE Tumbleweed:python38-Django-3.2.7-2.3",
						"openSUSE Tumbleweed:python39-Django-3.2.7-2.3"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 5.3,
						"baseSeverity": "MEDIUM",
						"vectorString": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:N/A:N",
						"version": "3.0"
					},
					"products": [
						"openSUSE Tumbleweed:python310-Django-4.2.11-2.1",
						"openSUSE Tumbleweed:python310-Django4-4.2.14-1.1",
						"openSUSE Tumbleweed:python311-Django-4.2.11-2.1",
						"openSUSE Tumbleweed:python311-Django4-4.2.14-1.1",
						"openSUSE Tumbleweed:python312-Django-4.2.11-2.1",
						"openSUSE Tumbleweed:python312-Django4-4.2.14-1.1",
						"openSUSE Tumbleweed:python36-Django-3.2.7-2.3",
						"openSUSE Tumbleweed:python38-Django-3.2.7-2.3",
						"openSUSE Tumbleweed:python39-Django-3.2.7-2.3"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2016-09-26T19:52:21Z",
					"details": "moderate"
				}
			],
			"title": "CVE-2018-16984"
		}
	]
}
